<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseManagerWidget</class>
 <widget class="QWidget" name="LicenseManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Active Licenses</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeView" name="gridLicenses">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
            <item>
             <widget class="QLabel" name="infoLabel">
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="detailsButton">
              <property name="text">
               <string>Selected License Details</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>New License</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QnLicenseWidget" name="licenseWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnLicenseWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/settings/license_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>gridSelectionChanged()</slot>
 </slots>
</ui>
